testkit/src/main/kotlin/com/autonomousapps/kit/GradleProject.kt

```diff
@@ -201,24 +192,20 @@ public class GradleProject(
         block(this)
       }
       androidSubprojectMap[name] = builder
+
+      return this
     }
 
     private fun defaultRootProjectBuilder(): RootProject.Builder {
       return RootProject.Builder().apply {
         variant = ":"
         gradleProperties = GradleProperties.minimalJvmProperties()
         settingsScript = SettingsScript()
-        buildScript = defaultRootProjectBuildScript()
+        buildScript = BuildScript()
         sources = emptyList()
       }
     }
 
-    private fun defaultRootProjectBuildScript(): BuildScript {
-      return BuildScript(
-        plugins = Plugins(listOf(Plugin.dependencyAnalysis, Plugin.kotlinPlugin(apply = false)))
```

Comment:
The included build projects should do this more centrally than how I've solved it currently.